frontend/pages/reports/index.js

```diff
@@ -0,0 +1,227 @@
+import s from 'styled-components'
+import Router from 'next/router'
+import { useEffect, useState } from 'react'
+import renderPage from '../../renderPage.js'
+import { Icon, Empty, Checkbox, CheckboxLabel } from '../../components/common'
+import { doApiRequest, API_BASE_URL } from '../../utils'
+import { Container } from '../../components/common/Container'
+import { CLUBS_GREY } from '../../constants/colors'
+import Edit from './edit'
+
+const GroupLabel = s.h4`
+  margin-bottom: 0em !important;
+  font-size: 32px;
+  color: #626572;
+`
+
+const ColoredHeader = s.div`
+  background: linear-gradient(to right, #ef4c5f, #4954f4);
+  height: 7em;
+`
+
+const TransparentButton = s.button`
+  width: 190px;
+  height: 35px;
+  border-radius: 17px;
+  border: 0;
+  background: rgba(255,255,255,0.32);
+  float: right;
+  font-size: 15px;
+  font-family: Lato;
```

Comment:
Is the font-family being switched to Lato in general? Feel strange to have one font in just one place

Reply:
Do not change font family. We are using`inter` per recent PR by @valenciafu .

Reply:
Oops my bad, thought it was Lato